### ISSUE TITLE
Hard bug! You can duplicate minegeld very fast! Dont use on servers!

### ISSUE THREAD

**Issue by wireva** [Tmanyo/bank_accounts#3]
just drag some money to the deposit slot and then you can move eg 5 mg notes over to the 10 field clieck enter and withdraw and boom, you have a bunch of fake money!!!!
Using contentDB version

**Comment by Tmanyo** [comment]
Thank you for bringing this to my attention.  This mod has not been actively developed in quite some time.  Knowing what I know now, I would rewrite this mod from scratch.  It is not well tested or written to be honest.  I would work on this issue, but I am very busy with my studies, and it is unlikely to ever be completed. Sorry

**Comment by ApolloX939** [comment]
Yeah my commit you merged fixes this too... but the main issue is the fact that I also introduced a dupe of items still.

(But at least we don't dupe money now)

In the Error Recovery sections we throw an error to the Minetest logs then return the item.

Some items instead of just returning it back, end up duplicating the item. (Not sure if we want to eat the item and just take it as a message to the user to stop cheating or if we want to continue returning the item rather duplication)

**Comment by Tmanyo** [comment]
I may look into this more in the next week, so I will leave this issue open until I've had a chance to go through the possible options.
